model_compression_toolkit/pytorch/pytorch_implementation.py

```diff
@@ -0,0 +1,283 @@
+        """
+        Apply shift negative correction (SNC) on a graph.
+
+        Args:
+            graph: Graph to apply SNC on.
+            qc: Quantization configuration.
+            fw_info: FrameworkInfo object with information about the specific framework's module.
+
+        Returns:
+            Graph after SNC.
+        """
+        return apply_shift_negative_correction(graph,
+                                               qc,
+                                               fw_info)
+
+    def attach_sc_to_node(self,
+                          node: BaseNode,
+                          output_channel_index: int) -> BaseStatsCollector:
+        """
+        Return a statistics collector that should be attached to a node's output
+        during statistics collection.
+
+        Args:
+            node: Node to return its collector.
+            output_channel_index: Index of output channels of layers in the model's framework.
+
+        Returns:
+            Statistics collector for the node.
+        """
+        return create_stats_collector_for_node(node,
+                                               output_channel_index=output_channel_index)
+
+    def get_substitutions_marking(self) -> List[common.BaseSubstitution]:
+        """
+
+        Returns: A list of the framework substitutions used for marking
+        points we fuse.
+
+        """
+        return [MarkActivation()]
+
+    def get_substitutions_pre_statistics_collection(self) -> List[common.BaseSubstitution]:
+        """
+
+        Returns: A list of the framework substitutions used before we build a quantized module.
+
+        """
+        return [BatchNormalizationFolding()]
+
+    def get_substitutions_post_statistics_collection(self,
+                                                     quant_config: QuantizationConfig) -> List[common.BaseSubstitution]:
+        """
+        Return a list of the framework substitutions used after we collect statistics.
+
+        Args:
+            quant_config: QuantizationConfig to determine which substitutions to return.
+
+        Returns:
+            A list of the framework substitutions used after we collect statistics.
+        """
+        substitutions_list = []
+        return substitutions_list
+
+    def get_substitutions_channel_equalization(self,
+                                               quant_config: QuantizationConfig,
+                                               fw_info: FrameworkInfo) -> List[common.BaseSubstitution]:
+        """
+        Return a list of the framework substitutions used for channel equalization.
+
+        Args:
+            quant_config: QuantizationConfig to determine which substitutions to return.
+            fw_info: FrameworkInfo object with information about the specific framework's module.
+
+        Returns:
+            A list of the framework substitutions used after we collect statistics.
+        """
+        substitutions_list = []
+        return substitutions_list
```

Comment:
We need to check if channel equalization is true and then we should raise exception

Reply:
We talk about it this function should not be implemented